aws/request/handlers.go

```diff
@@ -20,22 +20,6 @@ type Handlers struct {
 	AfterRetry       HandlerList
 }
 
-// Copy returns of this handler's lists.
-func (h *Handlers) Copy() Handlers {
```

Comment:
Just to note 3000ns is only 3us or 0.003ms not 3ms. Since requests can take multiple ms to cross the wire 0.003ms once per request is not significant.

Having user modifiable handlers is important, and sever users depend on this functionality today. Removing this would be a breaking change to the SDK.